pkg/manager-nnf/storage_group.go

```diff
@@ -131,8 +131,24 @@ func (sg *StorageGroup) Rollback(state uint32) error {
 				return err
 			}
 		}
+
+	case storageGroupDeleteStartLogEntryType:
```

Comment:
So this symbol has always been defined, but nothing has been acting on it until now?

Reply:
Yep. Oversight on my part - I thought the delete logic was smart enough to resume where it left off but it's not.